core/ui/src/main/java/com/twix/ui/image/ImageGenerator.kt

```
@@ -0,0 +1,75 @@
+    /**
+     * [Uri] 로부터 실제 [Bitmap] 을 디코딩한다.
+     *
+     * 새로운 InputStream을 열어 [BitmapFactory.decodeStream] 으로 변환한다.
+     */
+    private fun bitmap(
+        contentResolver: ContentResolver,
+        imageUri: Uri,
+    ): Bitmap =
+        contentResolver.openInputStream(imageUri)?.use { inputStream ->
+            BitmapFactory.decodeStream(inputStream)
+        } ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)